### PR TITLE
energy 1.0.10 (new formula)

### PR DESCRIPTION
**Pull request by sxmxta**
Add Energy CLI development environment tool

Energy CLI is a development environment for energy frameworks It can be installed, compiled, and packaged from the development environment

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

File: Formula/e/energy.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  test do
+    output = shell_output("#{bin}/energy cli -v 2>&1")
+    assert_match "Current", output
+    assert_match "Latest", output
+
+    output = shell_output("#{bin}/energy env 2>&1")
+    assert_match "Get ENERGY Framework Development Environment", output
+    assert_match "GOROOT", output
+    assert_match "ENERGY_HOME", output
```
Comment (chenrui333) [inline Formula/e/energy.rb:24]:
```suggestion
    output = shell_output("#{bin}/energy cli -v")
    assert_match "Current", output
    assert_match "Latest", output

    output = shell_output("#{bin}/energy env")
    assert_match "Get ENERGY Framework Development Environment", output
    assert_match "GOROOT", output
    assert_match "ENERGY_HOME", output
```

Comment (chenrui333) [reply Formula/e/energy.rb:24]: does your CLI has different version scheme?

```
$ energy cli -v
      GO
    ENERGY
$ CLI Current: 1.0.10
$ CLI Latest : 1.0.11
$ There new version available.
  Download: https://sourceforge.net/projects/energye/files/cli/energy-darwinarm-64.zip
```

Comment (chenrui333) [reply Formula/e/energy.rb:24]: it should be 1.0.1 rather than 1.0.10?

**Comment by sxmxta** [comment]
I really don't understand what this commit style is about

File: Formula/e/energy.rb (hunk 1 above)
Comment (sxmxta) [reply Formula/e/energy.rb]: The title has been modified to 1.0.10
Thank you, CLI is indeed 1.0.10. I didn't notice that it would be related to the title

**Review by sxmxta — COMMENTED** [review]
The code content has been changed as required

**Pull request closed without merging**